File: DemoServer/AdditionalLanguages/python/queries/fullCollectionQuery/fullCollectionQuery.py
from typing import List
from demo_example import Example
from models import Company

class FullCollectionQuery(Example):
    def run(self, run_params=None) -> List[Company]:
        #region Demo
        with self.document_store_holder.store().open_session() as session:
            #region Step_1
            full_collection_query = session.query(object_type=Company)
            #endregion

            #region Step_2
            collection_results = list(full_collection_query)
            #endregion
        #endregion

        return collection_results

File: DemoServer/AdditionalLanguages/php/queries/fullCollectionQuery/FullCollectionQuery.php
<?php

namespace RavenDB\Demo\queries\fullCollectionQuery;

//region Usings
//endregion

use RavenDB\Demo\common\DocumentStoreHolder;
use RavenDB\Demo\common\models\Company;

class FullCollectionQuery
{
    public function __invoke()
    {
        //region Demo
        $collectionResults = [];

        $session = DocumentStoreHolder::getStore()->openSession();
        
        try {
            //region Step_1
            $fullCollectionQuery = $session->query(Company::class);
            //endregion

            //region Step_2
            $collectionResults = $fullCollectionQuery->toList();
            //endregion
            
        } finally {
            $session->close();
        }
        //endregion

        return $collectionResults;
    }
}
